docs(Configuration): adding visual content and improve structure for brevity

DevTKSS opened the pull request:
Visual Content to simplify understanding possibilities (skip and list of skips can be tricky)
added information about help argument is available
Seperated CI topic to own area

## Done:

- [x] Will check if toc.yml needs future adjustments
- [x] Check added files to be linked properly
- [x] Enhanced Readme.md file so it can also be used as Intro

## TODO:

- [ ] Build has to show if the link to UnoCheck.Intro is successfull this way

## Help needed:

- Structure inside of Tabs should be unified similar to api docs with syntax, explaination incl. sample and to be checked synonym arguments, but this could also be done in syntax area before.

DevTKSS commented:
@agneszitte FYI: all checks passed here :)

Takoooooo commented on doc/configuration/configuring-uno-check.md:
```diff
@@ -2,55 +2,21 @@
+[!INCLUDES [Uno-Check-Windows-Elevation](../includes/uno-check-help.md)]
 
-### `--target` Choose target platforms
+# [**target platforms**](#tab/arg-target)
```
Can we instead of the descriptions use the name of the parameter instead everywhere? In that case [-t|--target]. Becuase to users that not familiar with these parameters just "Target platforms" or "Continuous Integration" won't tell anything. Lets make the parameter name the header of the tab and the context of the tab will describe what it does

DevTKSS replied:
will check on them and then add the screenshots taken here. but for those with the --version and --help I only get them with localized description, maybe that will then need someone with en-us localization to adjust this parts

DevTKSS replied:
@Takoooooo see https://github.com/unoplatform/uno.check/pull/362/commits/635d5941325e1d3b1f5706f4f0e5015fb1d58a0f
for applyed this requested change. I used just "target" because I expect this to be the argument from last time I was able to use `uno-check -h` with actually getting those options.

Takoooooo commented on doc/configuration/configuring-uno-check.md:
```diff
@@ -2,55 +2,21 @@
 
-## Command line arguments
+[!INCLUDES [uno-check-help](../includes/uno-check-help.md)]
 
-The following command line arguments can be used to customize the tool's behavior.
+[!INCLUDES [Uno-Check-Windows-Elevation](../includes/uno-check-help.md)]
```
Not sure why we need these? 
![image](https://github.com/user-attachments/assets/feba8b73-5a64-4574-a35c-fd0608b356bb)
For me it doesen't render and also it references the same link

DevTKSS replied:
@Takoooooo are those the build or vs code preview? seems like no extension is capable of rendering includes at this moment. try it with local build instead. I set up for myself a docfx with modern template (PR at main repo thats needs someone fix the last powershell location issue to hopefully than enable also non uno team contributors use the import external docs script to get their externals if some to check them) to validate those kind of things locally. as you will know, thats not optimal as workaround but I dont know any other way to get there realistically otherwise

DevTKSS commented:
@Takoooooo added the local images I meant already now, maybe you want to take a look and replace if yours is localized correctly so i could link them then later on

jeromelaban commented on doc/assets/uno-check-config-output.png:
Let's keep using english for the screenshots

DevTKSS replied:
@jeromelaban okay, then I will need someone to help for the screenshots, as my system is only german and until now I did not found an option to tell dotnet cli I want en-US loc output.

Takoooooo replied:
@DevTKSS you can simply "set DOTNET_CLI_UI_LANGUAGE=en" if you are on Windows

jeromelaban commented on doc/configuration/configuring-uno-check.md:
```diff
@@ -114,53 +111,87 @@ The manifest is hosted by default here: [uno.ui-preview.manifest.json](https://r
-### `--pre-major`, `--preview-major`
+**Synonym Arguments:** `--pre`, `--preview`, `-d`, `--dev`
+
+# [**pre-major**](#tab/pre-mayor)
```
```suggestion
# [**pre-major**](#tab/pre-major)

```

DevTKSS commented:
- [x] checked the INCLUDE links 
- [x] squashed for brevity to keep only the essential ones, squashing chore commits e.g. formatting
- [x] Added cases for tfm which seem to result in #404 as the desktop or winsdk flags are not known

To check: the tfm flag should normally not be equal to the target flag:

`--target` should stand for the current checked targets like desktop, windows, winsdk ios or android
`--tfm` should stand for the net version like `net9.0` or `net10.0` in the future or major-prerelease case

DevTKSS commented:
Seems like this is indeed failing just because of issue #417 While I did not even realize the CI would fail on this PR because of the version 😅
I am curious how that worked out before. 🤔

DevTKSS commented:
@Jen-Uno could you put this PR on the list for "needs assistance"? I am not able to get my uno check to speak english with me, while dotnet does. Updated the branch right now, but will be out of office for a week now just for your notes of my open PR's if I migth not be responding or not able to work on it

Jen-Uno commented:
will do! have a great vacation!
